doc/patch-author-guide.md

```diff
@@ -808,6 +808,41 @@ This can be done by replacing any usages of `static_branch_likely()`,
 `static_branch_unlikely()`, `static_key_true()`, and `static_key_false()` with
 `static_key_enabled()` in the patch file.
 
+There is an example  of using `static_key_enabled()` to replace the static key in the patch file:
+This is an static key used in the patch file:
+```
+static inline bool memcg_kmem_enabled(void)
+{
+    return static_branch_unlikely(&memcg_kmem_enabled_key);
+}
+``` 
+    
+We can use `static_key_enabled()` to rewrite the macros `static_branch_likely` as follows:
+```
+#define static_branch_unlikely(x) unlikely(static_key_enabled(&(x)->key))
+```
```

Comment:
IMHO (re)defining a kernel macro like static_branch_unlikely is a recipe for developer / review confusion.  I think the best practice would be substituting with `unlikely(static_key_enabled(&memcg_kmem_enabled_key))`

Reply:
Redefining a kernel macro is a choice if this macro effect limit functions. However, it is not a good idea to redifine it in most cases which is mentioned in the following text. Of course, I can rewrite this description in a more appropriate method.